.gitignore

```diff
@@ -21,3 +24,6 @@
 
 # Ignore Byebug History
 .byebug_history
+
+# Ignore Emacs stuff
+*~
```

Comment:
PS: realize it's after the fact now, but i think it's a totally on-point suggestion to keep this out of the project `.gitignore` and leave editor-specific configs to those with said editor. also: hi @latazzajones! nice to meet you! :)PS: realize it's after the fact now, but i think it's a totally on-point suggestion to keep this out of the project `.gitignore` and leave editor-specific configs to those with said editor. also: hi @latazzajones! nice to meet you! :)